src/style/sections/libraries/libraries-section.less

```diff
@@ -88,19 +88,27 @@
             justify-content: space-between;
             flex: none;
             height: 4rem;
-            margin: 0 2rem;
+            margin: 0 0rem;
             align-items: center;
-            border-top: @hairline solid @layer-divider;
-            border-bottom: @hairline solid @layer-divider;
         }
 
         &__content-input {
             border-bottom: @hairline solid @focus-highlight;
         }
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        flex: none;
+        height: 5rem;
+        margin: 0 2rem;
+        align-items: center;
+        
         .drop-down {
             flex-grow: 1;
             border: 0;
-
+            width: 18.4rem;
+            border-bottom: @hairline solid fade(@color-subsection-rule, 60%);
```

Comment:
Should we be `fade`ing colors here? Is that kosher? Is there a better way that will make our lives easier later when trying to maintain different color themes? CC @placegraphichere